### DISCUSSION TITLE
How frequently are language fixes pulled in to release?

### DISCUSSION THREAD

**Discussion by therealeldaria** [firefly-iii/firefly-iii#5865] [General]
I noticed that some translations for Swedish are very wrong, so I figured I would submit corrections, but noticed that someone already submitted them years ago. However in the GUI it is still wrong.

For example Bills are when set to Swedish "Noter" This is incorrect and should be "Räkningar"

Also under liabilities the translation for selections for Liability in/out is bonkers.
In English the choices are:
- I owe this debt to somebody else
- I am owed this debt

In Swedish it says:
- Jag är skyldig någon annan denna skuld
- Jag är skyldig denna skuld.

They both mean exactly the same thing, translated back to English they mean:
- I owe someone else this debt.
- I owe this debt.

There are a few other translation errors, however when I look at the translation software and in the source code it looks correct, ok I did not check all, but for example the one for Bills was corrected 3 years ago from what I can see in Crowdin.

**Comment by JC5** [comment]
To answer your question: Translations are updated during every release, so basically every month.

- ➡️  Could you link to the exact strings on Crowdin, please?
- ➡️  Could you point out *exactly* where it is wrong in Firefly III?

The latest translation is used by Crowdin, and downloaded by me. Sometimes however, translators "approve" a translation which means that new translations will not be used. It is also possible for people to vote for translations, which means that new translations do not get used.

Since I speak no Swedish I can't vouch for the quality unless people point out and fix errors, for which I'm very grateful. Tak!

**Reply by therealeldaria** [reply]
I'm native in Swedish, Almost native level in English and Danish.

The 'Bills' strings are in multiple locations. But here is one for example.
https://crowdin.com/translate/firefly-iii/138/en-sv#5372

And for the Liabilities.
https://crowdin.com/translate/firefly-iii/142/en-sv#11436
https://crowdin.com/translate/firefly-iii/142/en-sv#11438

I also noticed that some translations in Crowdin are wrong.
For example it looks like someone has used Google Translate for the "I am owed this debt" As Google Translate also get it wrong. I have added the correct translation, it can be tested by doing a reverse translation in Google Translate.

Also I noticed that someone has added the translation "Ansvar" for Liabilities. Although it could be translated to this it would not be correct in this context. The word "Ansvar" actually means Responsibility, and although correct that a Debt is a responsibility. In this context of a debt, the correct term would be "Skuld".

**Reply by therealeldaria** [reply]
Actually I will go through all strings in Crowdin and update or vote on the correct one if one is already done. ;-)

**Reply by JC5** [reply]
You can raise issues on Crowdin, that will alert me about changes.
